test/accelerators/NNPA/backend/CMakeLists.txt

```diff
@@ -93,7 +93,15 @@ endif()
 # Followings are test cases of test_to_enable_dict in test/backend/inference_backend.py.
 # Only test cases for operations supported by zDNN are listed. Non-supported cases are
 # commented out. This list is set in the environment variable `TEST_CASE_BY_USER`. So,
-# instruction name is added after test case name in each test case.
+# instruction name and dimParams are added after test case name if necessary.
+# For example, the following line
+#    test_add_cpu,zdnn_add,"0:0=a,1=b,2=c|1:0=a,1=b,2=c"
+# means that (1) the test name is "test_add_cpu", (2) instruction name is
+# "zdnn_add" to check the function is called, (3) when dimParams is
+# "NO_DYNAMIC_SHAPE_TEST", backend test is skipped, otherwise the string is
+# passed as --dimParams option. "0:0=a,1=b,2=c|1:0=a,1=b,2=c" means that the
+# first ranks of the first, second and third ranks of the first and second
```

Comment:
Could you make this explanation easier to follow? Don't use rank but dimension. Ranks in MLIR means the number of dimensions (in this case rank = 3).

Reply:
Thanks for the comments. I fixed. I confirmed that all "rank" were removed modifications by this PR.